scripts/west_commands/blobs.py

```diff
@@ -151,6 +153,23 @@ def fetch(self, args):
                 self.dbg('Blob {module}: {abspath} is up to date'.format(**blob))
                 continue
             self.inf('Fetching blob {module}: {abspath}'.format(**blob))
+
+            if blob['license-click-through']:
+                user_input = input("For this blob, need to read and accept license to continue."
+                                   "Read it? [y/n]: ")
```

Comment:
Splitting this over multiple lines will still concatenate it without spacing. Also not sure about the text itself.

```suggestion
                user_input = input("For this blob, need to read and accept license to continue. "
                                   "Read it? [y/n]: ")
```

Reply:
Thanks. Will change as this.